### PR TITLE
Log remix events

### PR DESCRIPTION
**Pull request by aninternetof**
~~The logging works and the tests pass, but it seems that the remix is resulting in a blank program. I see no failures in the Network tab of the Chrome debugger.~~

~~Any thoughts on how adding this log could be borking the remix flow?~~

This adds Sumo logging for a remix event.

### CODE REVIEW

**Comment by cabarnes** [comment]
No errors in the console?

**Comment by aninternetof** [comment]
No, no console errors. I'll keep playing with it, too.

**Comment by cabarnes** [comment]
Does the message get sent to Sumo? You should be able to set a breakpoint on that statement to make sure it's getting there

**Comment by aninternetof** [comment]
The message does show up in Sumo, which makes it a little stranger, even.

**Comment by cabarnes** [comment]
I'm thinking that it's because you aren't returning a `Promise` anymore. Try to add `return saveProgram...`

**Comment by aninternetof** [comment]
That did it! Thanks!

File: src/components/__tests__/Workspace.test.js [hunk 1]
```diff
@@ -19,10 +19,12 @@ import { COVERED, NOT_COVERED } from '@/actions/sensor';
 import { pushCommand } from '@/actions/rover';
 
 jest.mock('node-blockly/browser');
+jest.mock('sumo-logger');
```
Comment (cabarnes) [inline src/components/__tests__/Workspace.test.js]: There's a remix test that you could add an assertion to that `log` was called like the logger test does

**Pull request merged**